schema.graphql

```diff
@@ -152,3 +154,17 @@ type Transaction @entity {
   earnings: BigDecimal
   gysrSpent: BigDecimal
 }
+
+type Funding @entity {
+  # geyser id, created at timestamp
+  id: ID!
+
```

Comment:
can you also add a `token` field in here? it's not an important distinction with v1, but will become one in v2 with geysers setup to support multi-token

Reply:
Yep good idea. Added that in the new commit